lib/spack/spack/build_systems/python.py

```diff
@@ -41,7 +41,7 @@ def import_modules(self):
         in the source tarball directory. If the module names are incorrectly
         detected, this property can be overridden by the package.
 
-        Returns:
+        Return:
```

Comment:
Returns is correct

Reply:
```suggestion
        Returns:
```

Reply:
Yes, I just needed something to trigger unit tests